Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -158,6 +158,28 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
         .AddParameter("expression", _("Bottom Y position"))
         .AddParameter("expression", _("Chamfer (in pixels)"))
         .SetFunctionName("DrawChamferRectangle");
+
+
+  obj.AddAction("Torus",
+               _("Torus"),
+               _("Draw a torus on screen"),
+               _("Draw at _PARAM1_;_PARAM2_ a torus with inner radius:"
+               "_PARAM3_ and outer radius _PARAM4_ and "
+               "with start arc _PARAM5_ and end arc _PARAM6_"
+               "with _PARAM0_"),
```

Comment:
Even if the extension is not really consistent about this, there should be either no ":" or ":" everywhere. I wonder if using "of" would be better or not.
```suggestion
               _("Draw at _PARAM1_;_PARAM2_ a torus with inner radius"
               "_PARAM3_ and outer radius _PARAM4_ and "
               "with start arc _PARAM5_° and end arc _PARAM6_°"
               "with _PARAM0_"),
```